lib/chat/chat.dart

```diff
@@ -70,42 +71,13 @@ class _ChatState extends State<Chat> {
   }
 }
 
-buildHeader() {
-  return Container (
-    margin: EdgeInsets.all(30.0),
-    child: Column(
-      children: <Text>[
-        Text(
-          chatHeaderTitle, 
-          style: GoogleFonts.muli(
-            textStyle: TextStyle(
-              color: Colors.black, 
-              letterSpacing: .5, 
-              fontSize: 30.0, 
-              fontWeight: FontWeight.bold
-            )
-          )
-        ), 
-        Text(
-          chatHeaderSubtitle, 
-          style: GoogleFonts.muli(
-            textStyle: TextStyle(
-              color: Colors.grey, 
-              letterSpacing: .5, 
-              fontSize: 18, 
-              fontWeight: FontWeight.w600
-            )
-          )
-        ), 
-      ]
-    )
-  );
-}
-
 buildChatView(var newMatchesList, var matchesList) {
   return (
     <Widget>[
-      buildHeader(),
+      GlobalHeader(
+        actionText: "New Message",
+        onActionTap: () => {},
+      ),
       if (newMatchesList.length != 0)
         NewMatches(newMatchesList: newMatchesList),
       if (matchesList.length != 0)
```

Comment:
oh wow I didnt know we could do if statements without the brackets!

Reply:
If it's a one-liner thing, yes! Although I know in React, linters complains about it.